### PR TITLE
⬆️ Bump libplanet to 3.4.0

### PR DESCRIPTION
**Pull request by greymistcube**
⬆️

### CODE REVIEW

File: NineChronicles.Headless/NineChroniclesNodeService.cs [hunk 1]
```diff
@@ -277,8 +278,7 @@ internal void ConfigureContext(StandaloneContext standaloneContext)
             standaloneContext.Store = Store;
             standaloneContext.Swarm = Swarm;
             standaloneContext.CurrencyFactory =
-                new CurrencyFactory(
-                    () => standaloneContext.BlockChain.GetAccountState(standaloneContext.BlockChain.Tip.Hash));
+                new CurrencyFactory(() => standaloneContext.BlockChain.GetAccountState((BlockHash?)null));
```
Comment (moreal) [inline NineChronicles.Headless/NineChroniclesNodeService.cs:281]: I wonder about this change's intention 🤔. It seems the `CurrencyFactory` cannot access any states as my understanding. `BlockChain.GetAccountState -> BlockChainStates.GetAccountState -> BlockChainStates.GetTrie -> TrieStateStore.GetStateRoot`. Is there another change that I missed?

File: NineChronicles.Headless/GraphTypes/StandaloneQuery.cs [hunk 2]
```diff
@@ -120,33 +120,28 @@ blockHash switch
 
                     var recipient = context.GetArgument<Address?>("recipient");
 
-                    IEnumerable<Transaction> txs = digest.TxIds
-                        .Select(bytes => new TxId(bytes))
+                    IEnumerable<Transaction> blockTxs = digest.TxIds
+                        .Select(b => new TxId(b.ToBuilder().ToArray()))
```
Comment (moreal) [inline NineChronicles.Headless/GraphTypes/StandaloneQuery.cs:124]: Is it needed to become `byte[]` through the array builder? I wonder its benefit.

File: NineChronicles.Headless/NineChroniclesNodeService.cs (hunk 1 above)
Comment (riemannulus) [reply NineChronicles.Headless/NineChroniclesNodeService.cs:281]: Maybe it is just a mistake I guess..?

File: NineChronicles.Headless/ActionEvaluationPublisher.cs [hunk 3]
```diff
@@ -599,14 +591,14 @@ private bool ContainsAddressToBroadcast(ActionEvaluation<ActionBase> ev)
...
+                int t = ev.RandomSeed;
+                return true;
             }
 
             private bool ContainsAddressToBroadcastRemoteClient(ActionEvaluation<ActionBase> ev)
             {
-                var updatedAddresses = ev.OutputState.Delta.UpdatedAddresses;
-                return TargetAddresses.Any(updatedAddresses.Add(ev.Signer).Contains);
+                int r = ev.RandomSeed;
+                return true;
```
[the remark this answers is not in the document]
Comment (riemannulus) [reply NineChronicles.Headless/ActionEvaluationPublisher.cs:601]: Those methods are useless now, Replace by line 469.

**Review by moreal — APPROVED** [review]

**Pull request merged**